### ISSUE TITLE
Preview for on-chain tokens

### ISSUE THREAD

**Issue by ghost** [SmaugPool/pool.pm#53]
Hi, a couple of days ago previews for on-chain NFTs started displaying in static mode, but with overlapping color from frames on pool.pm (example at $tokenstakepool) 

Is it possible to display animated or if not, a single frame.
Thank you very much.

**Comment by SmaugPool** [comment]
Thank you for reporting.

For now pool.pm is displaying GIFs first frame for thumbnails encoded in webp. It's planned to show them animated again in a few weeks.

I investigated the color issue of the first frame and it's not due to "overlapping color from frames" but to webp compression.

https://pool.pm/asset1xc66nzqy246y2rvp4sw96gn6ydt5yq884h2yus

As an example, here is the first frame (resized to 128x128 without any interpolation to avoid browser resizing):
![image](https://user-images.githubusercontent.com/66848001/206440376-0532fe43-5105-480e-b39f-df2c260f598e.png)

Now here is the first frame saved as lossy webp using gimp (quality 90 but this does not really change the result) then resized to 128x128 without interpolation:
![image](https://user-images.githubusercontent.com/66848001/206440858-10f920fe-c671-41c8-a6c0-328574bc7dbe.png)

You can notice the same effect than displayed on pool.pm due to webp compression algorithm:
![image](https://user-images.githubusercontent.com/66848001/206441013-fecabafa-0ea7-42d9-a5fa-8aefdaef6cbd.png)

In conclusion, there is no frame overlapping bug, but an unfortunate effect of webp compression algorithm on pixel art.
The result should be back to your expectations in a few weeks (no ETA yet), I will let the issue open until then.

**Comment by SmaugPool** [comment]
The issue has been solved. 

For now the first frame is still used and optimized, but using a better algorithm to match as much as possible the original:

![image](https://user-images.githubusercontent.com/66848001/206668836-4bb869b3-06cd-467b-b403-5853fdc15021.png)

Please confirm if the result matches your expectations.
